src/features/interview/timer.tsx

```
@@ -22,23 +26,33 @@ type Props = {
+
   const characterId = useCharacterStore((state) => state.characterId);
   const { handleExperienceUp } = useExperienceUp();
 
-  const isFinalQuestionAsked = messageList.length >= 2 && messageList[1].role === 'assistant';
+  const questionIndex = useInterviewStore((state) => state.questionIndex);
+  const isFinalQuestionAsked = questionIndex >= INTERVIEW_LIMIT_COUNT;
+
+  {
+    /** TODO: 에러 처리 알림에 대한 고민 필요 */
+  }
+  if (InterviewHistoryError || aiFeedbackError) {
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)